src/components/pages/directory/content/blockquote/blockquote.jsx

```
@@ -0,0 +1,55 @@
+        <p className="text-pretty text-[28px] font-medium leading-normal tracking-tight md:text-2xl md:leading-normal sm:text-[20px]">
+          {quote}
+        </p>
+      </blockquote>
+      {avatars && avatars.length > 0 && (
+        <div className="mt-4 flex items-center gap-2 sm:mt-5">
+          {avatars.map((avatar) => (
+            <img
+              src={avatar}
+              alt={avatar}
+              className="size-7 rounded-full border-[2px] border-gray-8 sm:hidden"
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)